lib/core/config.rb

```diff
@@ -96,14 +100,7 @@ def apps
   def current
     return unless app
 
-    @current ||= begin
-      app_config = find_app_config(app)
-      ensure_config_app!(app, app_config)
-
-      warn_deprecated_options(app_config)
```

Comment:
Deprecated options were not being warned, because at this point, we've already modified old keys to new keys, so the deprecated keys need to be checked in `apps`.